aws/resource_aws_autoscaling_group_test.go

```diff
@@ -2069,6 +2069,41 @@ func TestAccAWSAutoScalingGroup_MixedInstancesPolicy_LaunchTemplate_Override_Wei
 	})
 }
 
+func TestAccAWSAutoScalingGroup_launchTempPartitionNum(t *testing.T) {
+	var group autoscaling.Group
+
+	randName := fmt.Sprintf("terraform-test-%s", acctest.RandString(10))
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:            func() { testAccPreCheck(t) },
+		Providers:           testAccProviders,
+		CheckDestroy:        testAccCheckAWSAutoScalingGroupDestroy,
+		DisableBinaryDriver: true,
```

Comment:
Nit: This temporary flag is only necessary for acceptance testing that is checking `TypeSet` values at the moment.